packages/react-native-web/src/exports/Image/index.js

```diff
@@ -332,24 +340,94 @@ const Image: React.AbstractComponent<
   );
 });
 
-Image.displayName = 'Image';
+/**
+ * This component handles specifically loading an image source with header
```

Comment:
Make make a note that this doesn't load images with headers passed in `defaultSource` so that's perfectly clear